resources/var/tmp/resources/init.groovy.d/scripts/init010NORMproxyconfiguration.groovy

```diff
@@ -1,69 +1,65 @@
-// Script to activate proxy settings in jenkins if set in etcd
+// Script to activate proxy settings
 
 import jenkins.model.*
 import groovy.transform.Field
-import groovy.json.JsonSlurper;
 
-def getValueFromEtcd(String key){
-	String ip = new File("/etc/ces/node_master").getText("UTF-8").trim();
-	URL url = new URL("http://${ip}:4001/v2/keys/${key}");
-	def json = new JsonSlurper().parseText(url.text)
-	return json.node.value
-}
+File sourceFile = new File("/var/lib/jenkins/init.groovy.d/lib/EcoSystem.groovy")
+Class groovyClass = new GroovyClassLoader(getClass().getClassLoader()).parseClass(sourceFile)
+ecoSystem = (GroovyObject) groovyClass.getDeclaredConstructor().newInstance()
```

Comment:
Is it possible to create a factory function within the scripts directory? Could we then use it within the files this way?

`doguCtl = new DoguCtlFactory().getInstance()` as mentioned [here](https://www.jmdawson.net/blog/2014/08/18/using-functions-from-one-groovy-script-in-another/)